Fix #3682

ff66theone opened the pull request:
Potential fix for #3682, declaration of `eeprom` moved to main chunk.

ff66theone committed:
Fix #3682

Potential fix for #3682, declaration of `eeprom` moved to main chunk.

asiekierka commented on src/main/resources/assets/opencomputers/loot/openos/bin/flash.lua:
```diff
@@ -53,8 +57,6 @@ local function writeRom()
   end
 
-  local eeprom = component.eeprom
-
```
This breaks the ability of an EEPROM to be inserted after starting the program, but before flashing.

ff66theone replied:
I moved the warning message to that function and a line that I added should fix this issue.

ff66theone committed:
Update flash.lua

Addressed an issue with flashing an EEPROM that would be replaced while the program is running.

asiekierka commented:
I made a slightly more future-proof fix.
